src/app/portlets/content-types/fields/content-type-fields-tab/content-type-fields-tab.component.scss

```diff
@@ -41,6 +41,8 @@ $tab-name-height: 32px;
     text-transform: uppercase;
     z-index: 1;
     margin-right: $basic-padding * 3;
+    overflow: hidden;
+    white-space: nowrap;
 
```

Comment:
`text-overflow: ellipsis;` because errase the content, so can be edited ( `contenteditable="true"` )